pom.xml

```diff
@@ -58,6 +58,7 @@
         <arquillian-managed>true</arquillian-managed>
         <jboss-cli.executable>./jboss-cli.sh</jboss-cli.executable>
         <keycloak.management.port>10090</keycloak.management.port>
+        <skipTests>true</skipTests>
```

Comment:
@martin-kanis I'm not sure if it's a good idea to skipTests by default for any project. What about create a separate profile for Spring? One idea you can try to have -Pwildfly-managed could be the default profile if nothing is specified, and if -Pspring-boot is specified the default profile is deactivated.

I understand what was your idea here, but I think would be better to try separate profiles.